src/workbookManager.ts

```diff
@@ -79,7 +154,10 @@ export class WorkbookManager {
             if (
                 properties.getAttribute("command") == "SELECT * FROM [Query1]"
             ) {
-                properties.parentElement?.setAttribute("refreshOnLoad", "1");
+                properties.parentElement?.setAttribute(
```

Comment:
Do we want to always set this property? why did you change it from only when needed to always?

Reply:
@ron-b this change was in your code, I'm not actually sure what did you solve here, can you please check this?

Reply:
@sbeih I believe this was a bug fix to address custom templates